### PR TITLE
An end to end demo exists for authn k8s with sidecar integration

### PR DESCRIPTION
**Pull request by jtuttle**
https://ca-il-jira.il.cyber-ark.com:8443/browse/CONJ-4844

### CODE REVIEW

File: 4_create_test_app_project.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+announce "Creating Test App project."
+
+if has_project "$TEST_APP_PROJECT_NAME"; then
+  echo "Project '$TEST_APP_PROJECT_NAME' exists, not going to create it."
+else
+  echo "Creating '$TEST_APP_PROJECT_NAME' project."
+  oc new-project $TEST_APP_PROJECT_NAME
+
+  # Must run as root to write cert keys to disk.
+  oc adm policy add-scc-to-user anyuid -z default
```
Comment (doodlesbykumbi) [inline 4_create_test_app_project.sh]: lets add a TODO to make sure this doesn't get swept under the rug. overprivileged service account!

File: 4_create_test_app_project.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  
+  # Permissions
+  oc policy add-role-to-user edit developer
```
Comment (doodlesbykumbi) [inline 4_create_test_app_project.sh]: this isn't necessary and can be taken out

File: 4_create_test_app_project.sh (hunk 1 above)
Comment (jtuttle) [reply 4_create_test_app_project.sh]: Fixed.

File: 4_create_test_app_project.sh (hunk 2 above)
Comment (jtuttle) [reply 4_create_test_app_project.sh]: Fixed.

**Pull request merged**